### PR TITLE
Marquee pride

### PR DESCRIPTION
**Pull request by khanRabiul**
### Marquee: 
**Path:** src > component > Header > Marqee.jsx
**Uses:** Into sSckyNavbar.jsx on the very top of the header marqee is added.
**Source:** Manually. There is no marquee or scroll animation available on material tailwind.
**NB:** **_To implement the animation I need to configure tailwind.config.js_**

### Pride Section:
**Path:**
       _**images:**_  src > assets > images > pride
       _**file:**_  src > components > pride > Pride.jsx, PrideCard.jsx and prideData.js
**Uses:** Into MainLayout.jsx just before the Footer.jsx
 **Source:**    Manually.

✅ _Requesting you to review the code_

### CODE REVIEW

File: src/Layout/MainLayout.jsx
Comment (Md-RonyAhmed) [inline src/Layout/MainLayout.jsx:1]: Please don't add any components to Layout. You should add  components in the home page

**Comment by Md-RonyAhmed** [comment]
You need to pull master branch updated code then do changes to your code

File: src/Layout/MainLayout.jsx
Comment (khanRabiul) [reply src/Layout/MainLayout.jsx:1]: I make the changes.

Comment (khanRabiul) [reply src/Layout/MainLayout.jsx:1]: But I can't send PR. It permit me to marge with the master branch. I need your help to solve the push problem.

**Pull request closed without merging**